test/rhsmlib_test/base.py

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env python
```

Comment:
Added.  Git Hub's not picking up the add because I added it below the she-bang line.